psi4/driver/procrouting/scf_proc/smart_scf.py

```diff
@@ -0,0 +1,168 @@
+from psi4.driver import p4util
+from psi4.driver import constants
+from psi4.driver.p4util.exceptions import ConvergenceError, ValidationError
+from psi4 import core
+
+
+class smart_solver():
```

Comment:
Classes in Python use CamelCase so `SmartSolver` or perhaps better `SmartSCFSolver`.